airflow/api/common/delete_dag.py

```diff
@@ -72,7 +72,7 @@ def delete_dag(dag_id: str, keep_records_in_log: bool = True, session: Session =
         )
     )
 
-    dags_to_delete = [dag_id for dag_id, in dags_to_delete_query]
+    dags_to_delete = [dag_id for (dag_id,) in dags_to_delete_query]
```

Comment:
This might be auto-generated? Is there a rule around it @jlaneve ?

Reply:
I thought ruff just explicitly turn into the tuple because not all cases could be resolved.

Like this one
```python
a = 42,
``` 

In could be a typo (more common problem) or just shorthand for 1-element tuple

Reply:
yeah this looks like it was a typo before - I've gone through and removed all cases like this

Reply:
For reference: https://docs.astral.sh/ruff/formatter/black/#single-element-tuples-are-always-parenthesized